qcfractal/cli/qcfractal_server.py

```diff
@@ -142,6 +142,10 @@ def parse_args():
     user_remove = user_subparsers.add_parser("remove", help="Remove a user.")
     user_remove.add_argument("username", default=None, type=str, help="The username to remove.")
 
+    # Dsahboard
+    user = subparsers.add_parser('dashboard', help="A Dashboard for the server (beta).")
```

Comment:
Change variable name from user to dashboard, or similar.

Reply:
Also probably say that the "dashboard" command causes the dashboard to be launched.